### PR TITLE
dataMultiBiz

### PR DESCRIPTION
**Pull request by reedhodder**
## Description of the change

> Add dataMultiBiz function that calls `data()` for each business core returned from `businessCores()`

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Cleanup
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Test

- [ ] Unit Test
- [ ] Tested in Dev/Local
- [ ] Tested in Prod

## Related issues

> Fix [#1]()

### CODE REVIEW

File: src/ParafinClient.ts [hunk 1]
```diff
@@ -80,14 +84,29 @@ class Client {
       .then(returnOrThrow)
   }
 
+  async dataMultiBiz(): Promise<Ok<ParafinResponse[], ParafinError>> {
```
Comment (elmurati) [inline src/ParafinClient.ts:87]: let's rename this two functions to:

```
// When we pass a business
async dataByBusiness(businessId?: string)

// When we want all businesses
async data()

```

File: src/ParafinClient.ts [hunk 2]
```diff
@@ -67,11 +70,12 @@ class Client {
     }
   }
 
-  async data(): Promise<Ok<ParafinResponse, ParafinError>> {
+  async data(businessId?: string): Promise<Ok<ParafinResponse, ParafinError>> {
```
Comment (elmurati) [inline src/ParafinClient.ts:73]: what happens when business id is null? do we default to the old behavior?

File: src/helpers/responseManager.ts [hunk 3]
```diff
@@ -63,17 +63,18 @@ function partnerResponse(
...
+  const results = baseResponse(businessCores)
+  const response: BusinessCoreResponse[] = []
 
   if (results != null && results.length > 0) {
-    response.businessId = results[0].id
-    response.externalId = results[0].external_id
+    results.map((bizCore: any) => {
+      response.push({
+        businessId: bizCore.id,
+        externalId: bizCore.external_id
```
Comment (elmurati) [inline src/helpers/responseManager.ts:75]: should we use a type for this?

Comment (reedhodder) [reply src/helpers/responseManager.ts:75]: all the other response managers use `any` at some point, probably fine for now

**Review by elmurati — APPROVED** [review]

**Pull request merged**